packages/lesswrong/components/sunshineDashboard/SunshineNewUsersItem.jsx

```diff
@@ -51,10 +53,15 @@ class SunshineNewUsersItem extends Component {
   }
 
   render () {
-    const { user, hover, anchorEl, classes } = this.props
+    const { user, hover, anchorEl, classes, currentUser } = this.props
 
     const { SunshineListItem, SidebarHoverOver, MetaInfo, SidebarActionMenu, SidebarAction, FormatDate, SunshineNewUserPostsList, SunshineNewUserCommentsList } = Components
 
+
+    const showNewUserContent = currentUser && currentUser.sunshineShowNewUserContent
+
+    if (this.state.hidden) { return null }
```

Comment:
Can you move this as early as possible into the function? Probably right after the props decomoposition